Source/Core/SMTLibOptions.cs

```diff
@@ -20,5 +20,13 @@ public interface SMTLibOptions
     bool SIBoolControlVC { get; }
     bool RestartProverPerVC { get; }
     bool Trace { get; }
+    
+    /**
+     * Setting this to true will rename all identifiers in the Boogie program to a generated name that does not depend on the original name.
+     * Discarding the original names is useful to prevent the solver input from changing when identifiers are renamed in the
+     * Boogie program, which prevents unexpected changes in solver output.
+     */
+    bool DiscardNames { get; }
```

Comment:
Is there any reason _not_ to always do this transformation? It seems good for everyone, for perhaps this should just always be on. (Or, if this causes the output to use the generated names, then I'm doubtful that anyone could afford to use it.)